addons/l10n_pk/data/account_tax_vat_report.xml

```diff
@@ -0,0 +1,790 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="l10n_pk_vat_form" model="account.report">
+        <field name="name">Tax Report Pakistan (PK)</field>
+        <field name="root_report_id" ref="account.generic_tax_report"/>
+        <field name="country_id" ref="base.pk"/>
+        <field name="filter_fiscal_position" eval="True"/>
+        <field name="availability_condition">country</field>
+        <field name="column_ids">
+            <record id="l10n_pk_vat_column_net" model="account.report.column">
+                <field name="name">Balance</field>
+                <field name="expression_label">balance</field>
+            </record>
+            <record id="l10n_pk_vat_column_tax" model="account.report.column">
+                <field name="name">Tax Amount</field>
+                <field name="expression_label">taxbalance</field>
+            </record>
+        </field>
+        <field name="line_ids">
+            <record id="l10n_pk_vat_title_purchases_vat" model="account.report.line">
+                <field name="name">Purchases Taxes</field>
+                <field name="hierarchy_level">0</field>
+                <field name="children_ids">
+                    <record id="l10n_pk_vat_line_SP_17" model="account.report.line">
+                        <field name="name">Standard Purchases Tax 17%</field>
+                        <field name="code">PUR_ST_17</field>
+                        <field name="expression_ids">
+                            <record id="l10n_pk_vat_expression_SP_0_balance" model="account.report.expression">
+                                <field name="label">balance</field>
+                                <field name="engine">tax_tags</field>
+                                <field name="formula">Standard Purchases Tax 17% (Base)</field>
+                            </record>
+                            <record id="l10n_pk_vat_expression_SP_0_tax" model="account.report.expression">
+                                <field name="label">taxbalance</field>
+                                <field name="engine">tax_tags</field>
+                                <field name="formula">Standard Purchases Tax 17% (Tax)</field>
```

Comment:
Yes it was a comment from me for another localization, but I actually find it cleaner how it's done here. Also, there are no real convention on how we should format tax tags. So unless the final reviewer thinks otherwise, I think you can keep it like that.